src/ir/bits.h

```diff
@@ -45,7 +45,7 @@ struct Bits {
     }
     // otherwise, see if adding one turns this into a 1-bit thing, 00011111 + 1
     // => 00100000
-    if (PopCount(mask + 1) != 1) {
+    if (mask & (mask + 1)) {
```

Comment:
`mask & (mask + 1)` is not fully equivalent of `PopCount(mask + 1) != 1` because not handle case when `mask == uint32_t(-1)` but this already checked [earlier](https://github.com/WebAssembly/binaryen/pull/2944/files#diff-58b7cf13906e9b5a4ba6306aaea394aaL40) so we cant use more simplify version without this special case here

Reply:
Makes sense to me. It would be good to fix the comment to reflect the new calculation, since it currently suggests that we are using popcount.